src/Templates/Factions/RHS_USMC_D.sqf

```diff
@@ -0,0 +1,210 @@
+/*
+RHSUSAF: USMC (W) templates for ARMA III
+*/
+
+_array = [];
+
+_array set [T_SIZE-1, nil];									//Make an array having the size equal to the number of categories first
+
+// Name, description, faction, addons, etc
+_array set [T_NAME, "tRHS_USMC (D)"];
+_array set [T_DESCRIPTION, "RHS USMC (D) units."];
```

Comment:
Maybe we should write that D stands for Desert camouflage either in the name or in the description? What do you think? If I saw this, I wouldn't understand what `USMS (D)` means, I think.